docs/OnnxReleases.md

```diff
@@ -9,120 +9,203 @@ SPDX-License-Identifier: Apache-2.0
+    1. Resolve the build issue
+    1. Submit a replacement PR based on your branch
 
-**Announce**
-* Announce in slack, for instance, `onnx-general` channel.
-* Notify ONNX partners like converter team and runtime team.
-* Create a news by updating `js/news.json` to announce ONNX release under [onnx/onnx.github.io](https://github.com/onnx/onnx.github.io) repo. For instance: https://github.com/onnx/onnx.github.io/pull/83.
+    * Example: https://github.com/conda-forge/onnx-feedstock/pull/116
 
-**Update conda-forge package with the new ONNX version**
-* Conda builds of ONNX are done via conda-forge, which runs infrastructure for building packages and uploading them to conda-forge. If it does not happen automatically, you need to submit a PR to https://github.com/conda-forge/onnx-feedstock (see https://github.com/conda-forge/onnx-feedstock/pull/1/files or https://github.com/conda-forge/onnx-feedstock/pull/50/files for example PRs) You will need to have uploaded to PyPI already, and update the version number and tarball hash of the PyPI uploaded tarball.
+* If the automatic PR is not created, you need to submit a PR manually
+    * Example: https://github.com/conda-forge/onnx-feedstock/pull/50
+    * Note: Use the sha256 hash (`sha256sum onnx-X.Y.Z.tar.gz`) of the release's tar.gz file from https://github.com/onnx/onnx/releases.
 
 **Merge into main branch**
-* After everything above is done, merge the release branch into the main branch to make it consistent. This step is needed only when there are urgent changes that are made directly into the release branch. The main branch does not have these needed changes. In all other circumstances, the merge PR shall show as empty so nothing needs to be merged.
+* If urgent changes were made directly into the release branch, merge the release branch back into main branch.
+* If all PRs merged into the release branch (after it was cut) were cherry-picks from main, the merge PR will show as empty and this step is not needed.
 
 **Remove old onnx-weekly packages on PyPI**
-* Once ONNX has been released on PyPI, remove all previous versions of [onnx-weekly package](https://pypi.org/project/onnx-weekly/#history) on PyPI to save space.
-* Steps: Login and go [here](https://pypi.org/manage/project/onnx-weekly/releases/) -> Choose target package -> Options -> Delete.
-
-**Bump opset version for ai.onnx**
-* Bump opset version for ai.onnx domain in `onnx/defs/operator_sets.h` and `onnx/defs/schema.h` for use by future operator additions and changes. For example, this [demo PR](https://github.com/onnx/onnx/pull/4134/files).
-
-**Update IR TBD date if there is an IR bump in the release**
-* Update the latest IR TBD date in https://github.com/onnx/onnx/blob/main/onnx/onnx.in.proto and regenerate corresponding proto files in the main branch if there is an IR bump in the release.
+* Remove all [onnx-weekly packages](https://pypi.org/project/onnx-weekly/#history) from PyPI for the just released version to save space.
```

Comment:
Unfortunately this broke https://github.com/microsoft/onnxscript/issues/1318. But that's ok :)